### DISCUSSION TITLE
Chatbox resets to top after every output.

### DISCUSSION THREAD

**Discussion by Sonic2444** [oobabooga/text-generation-webui#3628] [Q&A] [answered]
Hi, I've recently ran into an issue where the chatbox resets to the very top (oldest/first prompt) after generating responses. It's a very recent issue that's started a few days ago, after some months now of regular use. 

Until this began happening the model would respond and show "Is typing..." and then its generated text would appear, with everything aligned to the bottom. Now after every input I enter, the scroll bar instantly resets to the very top, and then I need to scroll all the way down the chat log to see and continue the exchange. It makes for a cumbersome experience. Has this happened to anyone else, and if so is there a fix or edit that can be easily made? Perhaps it's something in /css? Thank you.

**Comment by missionfloyd** [comment, marked as answer]
It's _supposed_ to scroll to the bottom (#3601).

To have it bottom-aligned, change this to `column-reverse`.
https://github.com/oobabooga/text-generation-webui/blob/1cae78476159f3688408113ce3634290ceed508c/css/main.css#L272

**Reply by Sonic2444** [reply]
Thank you, that was the exact edit I needed.

**Reply by oobabooga** [reply]
I think that this should be fixed after this change:

https://github.com/oobabooga/text-generation-webui/commit/0986868b1b4d1115534917d187a3b2c03ab042e2
